blog-pipeline/tests/test_llm_providers.py

```
@@ -14,38 +14,25 @@
+_fake_config_mod.pipeline_config = MagicMock()
+sys.modules["blog_runner.config"] = _fake_config_mod
+
+from blog_runner.llm import (
+    _BASE_DELAY_S,
+    _MAX_RETRIES,
+    _TRANSIENT_STATUS_CODES,
+    _retry_on_transient,
+    call_anthropic,
+    call_gemini,
+    call_openai_compat,
+)
```

Ruff (0.15.9)

[error] Module level import not at top of file

(E402)